src/chunk.c

```diff
@@ -1081,7 +1082,7 @@ chunks_typecheck_and_find_all_in_range_limit(Hyperspace *hs,
 											 int limit,
 											 MemoryContext multi_call_memory_ctx,
 											 char *caller_name,
-											 long *num_found)
+											 uint64 *num_found)
```

Comment:
why are we using uint64 here and not just int?

Reply:
Ugh because we need to set `max_calls` in `FuncCallContext`, which is how we tell the SRF how many chunks to iterate through. `max_calls` has type `uint64`

Reply:
Ugh yeah it's because we need to set `max_calls` in `FuncCallContext`, which is how we tell the SRF how many chunks to iterate over in the result. `max_calls` is of type `uint64`